### PR TITLE
UI builder: AXE core integration

### PR DESCRIPTION
**Pull request by vyhnalekl**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Using the code created during hackathon (https://github.com/microsoft/fluentui/tree/hackathon/ui-builder-a11y). 
Added AXE core a11y checks. Errors are displayed in right side menu when component is selected.

### CODE REVIEW

File: packages/fluentui/react-builder/src/components/Designer.tsx [hunk 1]
```diff
@@ -342,6 +344,43 @@ export const Designer: React.FunctionComponent = () => {
 
   const [{ mode, isExpanding, isSelecting }, setMode] = useMode();
   const [showJSONTree, handleShowJSONTreeChange] = React.useState(false);
+  const [axeErrors, setAxeErrors] = React.useState([]);
+
+  React.useEffect(() => {
+    if (state.selectedJSONTreeElementUuid) {
```
Comment (ling1726) [inline packages/fluentui/react-builder/src/components/Designer.tsx:350]: you have quite a lot of logic under this guard, would it make sense to invert the condition and return early ?

Comment (miroslavstastny) [reply packages/fluentui/react-builder/src/components/Designer.tsx:350]: I would even leave the if here and move the rest to a separate function.

**Review by miroslavstastny — APPROVED** [review]

File: packages/charting/src/components/Legends/Legends.styles.ts [hunk 2]
```diff
@@ -94,7 +94,7 @@ export const getStyles = (props: ILegendStyleProps): ILegendsStyles => {
         ...fonts.small,
         lineHeight: '16px',
         padding: '8px',
-      }
+      },
```
Comment (vyhnalekl) [inline packages/charting/src/components/Legends/Legends.styles.ts:97]:
```suggestion
      }
```

File: packages/charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx [hunk 3]
```diff
@@ -132,7 +132,7 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
         className={this._classNames.root}
       >
         <FocusZone direction={FocusZoneDirection.horizontal}>
-          <svg width={svgDimensions.width} height={svgDimensions.height} style={{display: "block"}}>
+          <svg width={svgDimensions.width} height={svgDimensions.height} style={{ display: 'block' }}>
```
Comment (vyhnalekl) [inline packages/charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx:135]:
```suggestion
          <svg width={svgDimensions.width} height={svgDimensions.height} style={{display: "block"}}>
```

**Review by jurokapsiar — APPROVED** [review]

File: packages/fluentui/react-builder/src/components/Designer.tsx [hunk 4]
```diff
@@ -875,25 +874,7 @@ export const Designer: React.FunctionComponent = () => {
...
-                  {axeErrors.map(error => (
-                    <li>
-                      <strong>AXE</strong>&nbsp;
-                      {error}
-                    </li>
-                  ))}
-                </ul>
-              </div>
-            )}
+            {axeErrors.length > 0 && <ErrorPanel axeErrors={axeErrors} />}
```
Comment (assuncaocharles) [inline packages/fluentui/react-builder/src/components/Designer.tsx:877]: nit

```suggestion
            {axeErrors.length && <ErrorPanel axeErrors={axeErrors} />}
```

**Review by xugao — APPROVED** [review]

**Comment by jurokapsiar** [comment]
failing screener test is most probably a random/flaky issue. agreed with @xugao to proceed with merging

**Pull request merged**